js/routing.js

```diff
@@ -1,51 +1,51 @@
-var config = noddityConfig
 var EventEmitter = require('events').EventEmitter
 var makeRouter = require('hash-brown-router')
+var config = window.noddityConfig
 
-module.exports = function() {
+module.exports = function(getPost) {
```

Comment:
It looks like this `getPost` argument isn't used - it could be dropped from here and no longer passed in in the `mainViewModel` I think